### PR TITLE
Update Angular 11 + peer deps

### LINKED ISSUE #21: Please update it to avoid warnings about peer dependencies
Installing it with Angular 9.x gives me the following npm warnings:

npm WARN ngx-script-loader@1.3.0 requires a peer of @angular/common@>=6.0.0 <9.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN ngx-script-loader@1.3.0 requires a peer of @angular/core@>=6.0.0 <9.0.0 but none is installed. You must install peer dependencies yourself.

### PR DESCRIPTION
**Pull request by bartholomej**
Support for Angular11 and fix this annoying warning:

> ngx-script-loader@1.3.0" has incorrect peer dependency "@angular/common@^6.0.0 || ^9.0.0".

FIX #21 

Tested:
- [x] Your built-in tests passed succefully
- [x] Used with ng11 app
- [ ] Tested with ng7, ng8, ng9, ng10

I am not sure about compatibility with older Angular. 
There have been big breaking changes between 9 – 11 and also TypeScript was upgraded from 3.5 to 4.1.

Maybe you can try to release the `@next` version and play with it on SkatckBlitz ;)

### CODE REVIEW

**Comment by muratcorlu** [comment]
Thanks for PR! We need to update deploy script to be able to have a `next` release. I use semantic-release. I need to update it and probably create a new branch for a beta release.

I'm also curious if there is an automated way to test library with multiple angular versions.

I'll try to check this tomorrow unless you check it :)

**Comment by brendanowens** [comment]
Would love to see this merged in!

**Comment by muratcorlu** [comment]
Ah sorry. I'll check this today and merge it.

File: projects/ngx-script-loader/package.json [hunk 1]
```diff
@@ -14,8 +14,11 @@
     "url": "https://github.com/muratcorlu/ngx-script-loader/issues"
   },
   "homepage": "https://github.com/muratcorlu/ngx-script-loader#readme",
+  "dependencies": {
+    "tslib": "^2.0.0"
+  },
   "peerDependencies": {
-    "@angular/common": ">=6.0.0 <9.0.0",
-    "@angular/core": ">=6.0.0 <9.0.0"
+    "@angular/common": ">=6.0.0 <12.0.0",
```
Comment (muratcorlu) [inline projects/ngx-script-loader/package.json:21]: Now I'm considering to write here just `">=11.0.0"` with a major version release of ngx-script-loader. So people who want to use with an older version of angular can continue with current version and in every major version, we can just support latest version of Angular. What do you think?

**Pull request merged**